src/luascript.cpp

```diff
@@ -8795,8 +8810,7 @@ int LuaScriptInterface::luaPlayerSetStamina(lua_State* L)
 	uint16_t stamina = getNumber<uint16_t>(L, 2);
 	Player* player = getUserdata<Player>(L, 1);
 	if (player) {
-		player->staminaMinutes = std::min<uint16_t>(2520, stamina);
-		player->sendStats();
+		player->setStamina(stamina);
```

Comment:
what is this change? I dom't think it's related to prey system?

Reply:
Yes, it is, you can check setStamina function and there a few lines changed.